test/python/WMCore_t/Services_t/CRIC_t/CRIC_t.py

```diff
@@ -35,13 +35,13 @@ def testConfig(self):
         self.assertEqual(self.myCRIC['cacheduration'], 1)
         self.assertEqual(self.myCRIC['accept_type'], 'application/json')
         self.assertEqual(self.myCRIC['content_type'], 'application/json')
-        self.assertEqual(self.myCRIC['requests'].pycurl, None)
+        self.assertEqual(self.myCRIC['requests'].pycurl, True)
 
-        newParams = {"cacheduration": 100, "pycurl": True}
+        newParams = {"cacheduration": 100, "pycurl": False}
```

Comment:
I'm not sure why you changed here from True to False, just double check the logic.

Reply:
It's just a test. The default config has it `True`, so I test a non-default config where it's False